apps/docs/content/guides/database/full-text-search.mdx

```diff
@@ -1142,6 +1215,323 @@ data = client.from_('books').select().text_search('description', "'big' & !'litt
 </$Show>
 </Tabs>
 
+## Ranking search results [#ranking]
+
+Postgres provides ranking functions to sort search results by relevance, helping you present the most relevant matches first. Since ranking functions need to be computed server-side, we'll use RPC functions and generated columns.
```

Comment:
```suggestion
Postgres provides ranking functions to sort search results by relevance, helping you present the most relevant matches first. Since ranking functions need to be computed server-side, use RPC functions and generated columns.
```